docs/docs/sample-scripts/teams/export-teams-conversations.md

```diff
@@ -0,0 +1,99 @@
+# Export conversations from Microsoft Teams Channels
+
+Author: [Joseph Velliah](https://sprider.blog/export-conversations-from-microsoft-teams)
+
+## Problem Statement
+
+We utilize Teams during incidents and create channels for each. We would like to be able to export conversation history.
+
+- Right now the only option we have is to go through Security & Compliance.
+- Teams usage is growing in every organization and it would soon become unreasonably to only have Administrators be the ones doing exports of channels for all the Teams.
+
+### Solution
+
+This script uses CLI for Microsoft 365 to export the conversations from Microsoft Teams Channels. You don't need to be a tenant admin to export conversations but you still can only export conversations of Teams of which you are member.
+
+!!! notes
```

Comment:
Let's change it to `attention` to keep it consistent with how we communicate preview APIs in command's docs